pkgs/development/tools/jq/default.nix

```diff
@@ -9,6 +9,8 @@ stdenv.mkDerivation rec {
     sha256="0g29kyz4ykasdcrb0zmbrp2jqs9kv1wz9swx849i2d1ncknbzln4";
   };
 
+  outputs = [ "bin" "doc" "man" "dev" "lib" "out" ];
```

Comment:
This should be `[ "out" "doc" "man" "dev" "lib" ]`, changing the default output is confusing and out is empty.

Reply:
> changing the default output is confusing and out is empty.

Yeah, because the binaries should go to `bin`. I’d leave out `out`, if nixpkgs actually supported that.